MANIFEST.in

```diff
@@ -1,3 +1,4 @@
-include README.md LICENSE Pipfile Pipfile.lock
+include README.md LICENSE HISTORY.md AUTHORS.md Pipfile mkdocs.yml tox.ini pytest.ini strategy.ini
 
-recursive-include aea *
+recursive-include docs *
+recursive-include tests *
```

Comment:
`black` checks from `tox` fail because `examples`, `packages` and `scripts` are not included.

Reply:
for analogous reasons, `mypy` and `liccheck` fail.